app/jobs/waste_notification_job.rb

```diff
@@ -0,0 +1,23 @@
+class WasteNotificationJob
+  def self.perform(waste_registration_id, waste_pickup_time_id)
+    waste_pickup_time = Waste::PickUpTime.find_by(id: waste_pickup_time_id)
+    registration_to_check = Waste::DeviceRegistration.find_by(id: waste_registration_id)
+    waste_types = JSON.parse(StaticContent.find_by(name: "wasteTypes").content)
+
+    device = registration_to_check.notification_device
+    return unless device.present?
+
+    # Message to send
+    title = "SmartVillageApp - Abfallkalender"
+    body = "Am #{waste_pickup_time.pickup_date} wird #{waste_types[registration_to_check.notify_for_waste_type]["label"]} in #{registration_to_check.street} abgeholt"
```

Comment:
Das wording hier sagt mir noch nicht zu. 
Siehe Screenshot einer Meldung: 
![Bildschirmfoto 2021-03-05 um 12 55 44](https://user-images.githubusercontent.com/1942953/110112515-24347380-7db2-11eb-9481-d35ed6c8d89d.png)

- [ ] Datumsformat sollte dd.mm.yyyy sein
- [ ] "...in...Weg" Rechtschreibung passt nicht und es wird bestimmt noch andere Fälle geben, wo das so ist

Reply:
geändert in:

`Großmutzer Weg: Am 2021-03-10 wird Papier abgeholt`